src/types.js

```diff
@@ -201,6 +201,9 @@ export type Outbox = {|
   subject: string,
   timestamp: number,
   type: 'stream' | 'private',
+
+  /** If true, we don't send the message to the server. */
+  deferSending?: boolean,
```

Comment:
What's the difference in meaning between a missing value and `false`?

If there is no difference in meaning, then the property should just be always present. We'll want a migration to add it to existing outbox objects, as `false`.